### ISSUE TITLE
Editable Places?

### ISSUE THREAD

**Issue by jseppi** [bsudekum/MapTab#7]
Hey @bsudekum, I was thinking it would be nice if the list of places were editable. My own use case was wanting to have Austin be one of the random maps, but it is not in https://github.com/mapbox/gazetteer and I'm not sure you all want people adding their own pet places to that project.

I'd be happy to draft a pull request with an implementation if you think it is a reasonable idea. Screenshots below show what I'm thinking.

![image](https://cloud.githubusercontent.com/assets/697848/7615231/d3f1f1e8-f961-11e4-9cd3-d4c8f550b35d.png)

![image](https://cloud.githubusercontent.com/assets/697848/7615232/d8010008-f961-11e4-8caf-ae7b945fa14e.png)

Perhaps editing JSON is too much though. A simpler option could be to have a button to add current map location to places.

**Comment by bsudekum** [comment]
Yeah this looks good. For reference, places are coming from https://github.com/mapbox/gazetteer.

Some thoughts:
- I wonder if a csv style delimitation would be easier for people to read and to add to

``` csv
lat, lng
0, 0
20, 30
50, 60
20, 20
```
- Place name is not required, and actually should not be included
- We might want to put a limit on how many people can add especially if this is going to stored in a cookie.

**Comment by jseppi** [comment]
Cool, thanks for the feedback. I agree simple Lat, Lng would be better.
Maybe zoom as well?
I'll get to putting it together.
